api/src/main/java/io/strimzi/api/kafka/model/connect/build/Artifact.java

```diff
@@ -36,11 +37,12 @@
 
     public static final String TYPE_JAR = "jar";
     public static final String TYPE_TGZ = "tgz";
+    public static final String TYPE_ZIP = "zip";
 
     private Map<String, Object> additionalProperties = new HashMap<>(0);
 
     @Description("Artifact type. " +
-            "Currently, the supported artifact types are `tgz` and `jar`.")
+            "Currently, the supported artifact types are `tgz`, `jar` and `zip`.")
```

Comment:
I wonder if @PaulRMellor would ask for a coma before the `and`.

Reply:
IBM style wants a comma before conjunction in a series of three or more :+1:

Reply:
Thanks for the review. @im-konge is working on the STs in the https://github.com/strimzi/strimzi-kafka-operator/pull/4283 
I will talk to him, whether it makes sense to add `zip` tests into his PR (eventually after merging this).